TreasureData/TreasureData.h

```diff
@@ -392,6 +405,25 @@ typedef void (^ErrorHandler)(NSString* _Nonnull errorCode, NSString* _Nullable e
  */
 - (BOOL)isInAppPurchaseEventEnabled;
 
+#pragma mark - Personalization API
+
+/**
+ * Fetch user segments from cdp endpoint. Callback with either a JSON serialized object or an error.
+ * @discussion This will make a call to shared instance's cdpEndpoint. Make sure you configure cdpEndpoint before using this method
+ * @param audienceTokens List of audience tokens. There must be at least one token.
+ * @param keys Key value dictionary with at least user_id and td_client_id are required.
+ * @param options Request options. For possible options, see TDRequestOptionsKey.
+ * @param handler Completion callback with either JSON object or an error.
```

Comment:
Would be nice if we explicitly tell here the handler will be invoked on the caller queue.